server/controllers/profile.js

```diff
@@ -0,0 +1,53 @@
+const Profile = require("../models/ProfileModel");
+
+// create new profile using given arguments
+const addProfile = async (props) => {
+  let newProfile = await new Profile(props);
+  let profile = await newProfile.save();
+  return profile;
+};
+exports.createProfile = async (req, res, next) => {
+  try {
+    let response = await addProfile(req.body);
+    res.status(201).json(response);
+  } catch (err) {
+    next(err);
+  }
+};
+
+// updating existing profile using given id and new data
+const changeProfile = async (id, newData) => {
+  await Profile.findByIdAndUpdate(id, newData);
+};
```

Comment:
You can move this functionality to `updateProfile`

Reply:
I think also for the giving parameters.
so please tell me how?